### DISCUSSION TITLE
Listening for Livewire events in Alpine

### DISCUSSION THREAD

**Discussion by DonCamillo11** [alpinejs/alpine#4426] [1. Help]
I’m trying to listen for an event like it is documented on the livewire doc
https://livewire.laravel.com/docs/events#listening-for-livewire-events-in-alpine

`<div x-on:post-created.window="..."></div>`

It is dispatched from livewire with
`$this->dispatch('post-created');`

But this doesn’t work. Am I doing something wrong?

**Comment by ekwoka** [comment]
should work just fine.

Does the event bubble by default?

Does setting a normal event listener catch it?

is the div with x-on inside a x-data?
